pkg/operator/ceph/cluster/mon/spec.go

```diff
@@ -38,6 +38,14 @@ func ClusterNameEnvVar(name string) v1.EnvVar {
 	return v1.EnvVar{Name: "ROOK_CLUSTER_NAME", Value: name}
 }
 
+func ClusterDaemonEnvVars() []v1.EnvVar {
```

Comment:
These could be useful for other storage providers orchestrated by rook. How about if you move this func to the `k8sutil` package?